_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,158 @@
+* Data Warehouse workload was focused on data science or machine learning
+* There is a business requirement to stream data to the warehouse in near realtime
+
+I would consider **RedShift** when any of the following apply
+* I have a fairly constant and stable utilization profile that is well suited to pre-purchasing reserved instances for cost optimization
+* I want to to provide a unified data warehouse access layer comprised of federated AWS sources (e.g. operational RDS databases, S3) in order to build out native RedShift data marts
+
+**For most other use cases, I would favour Snowflake**.
+
+You may notice that we haven't discussed Amazon Redshift too much in our consideration of Cloud Data Warehouse solutions; we have and continue to work with customers using RedShift and are happy to continue doing so. Yet there are reasons why we often reach for other solutions:
+
+* RedShift has struggled to keep pace with the feature sets of competitors such as BigQuery and Snowflake
+* It is generally more engineering intensive, considering administration, utilisation and optimization - managing infrastructure is still a customer concern
+* Storage and Compute separation is primitive - whilst this is improving with recent RA3 nodes, the price premium to be paid is a disincentive, for something that still doesn't match what competitors are offering (true on-demand usage)
+* Lack of high availability - RedShift clusters are limited to operating in a single availability zone. Whilst a cluster can generally recover simply from individual data node failures, a power loss to a whole AZ would require customers to create a new cluster from snapshots in a different AZ
+
+
+At the end of the day, it's what your organisation is capable of supporting that is more important, which is why I also rate this article on the [Two Philosophies of Cost in Data Analytics](https://www.holistics.io/blog/the-two-philosophies-of-cost-in-data-engineering/), discussing the trade off between service cost and labour cost.
+
+
+## Where does this leave specialists of existing data warehouses?
+
+Many important questions arise when people consider migrating to Cloud Data Warehouses, such as
+
+* How to transition skilled database administrators and operations staff to cloud?
+* What extra skills are required with a Cloud Data Warehouses?
+* Is a Cloud Data Warehouse going to lower our TCO for Data Warehousing?
+
+These are complicated questions to answer but it's worth bearing in mind that the effort required to maintain business services doesn't go away, it just changes in nature. As we'll explore in the next article, the core skills of ETL developers are absolutely transferable.
+
+Some new themes emerge with Cloud Data Warehouses; though the technology or implementations may change, the concerns and risk controls will typically be very familiar to experienced data warehouse operations professionals.
+
+### FinOps
```

Comment:
This doesn't fit here. Doesn't relate to the above section. Feels lost in the article. Adrift. Abandoned. 
This and the next section feel unfinished. You pose a bunch of questions but don't really tell me why I should care.
They need more work, and probably need to be higher up the article, possible even above resiliency etc.